plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java

```diff
@@ -67,7 +67,6 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
             case SUPPORTS_COMMENT_ON_TABLE:
```

Comment:
You could support `COMMENT ON TABLE` in this PR.

Reply:
done

Reply:
verified the relevant tests as well